modules/testserver/src/main/java/io/swagger/assert4j/testserver/execution/ProjectExecutor.java

```diff
@@ -121,7 +121,7 @@ private TestServerExecution doExecuteProjectFromRepository(RepositoryProjectExec
     private TestServerExecution doExecuteProject(ProjectExecutionRequest projectExecutionRequest, boolean async) throws ApiException {
         try {
             TestServerExecution execution = testServerClient.postProject(projectExecutionRequest, async);
-            cancelExecutionAndThrowExceptionIfPendingDueToMissingClientCertificate(execution.getCurrentReport(), null);
+            // cancelExecutionAndThrowExceptionIfPendingDueToMissingClientCertificate(execution.getCurrentReport(), null);
```

Comment:
Introducing an enum so you can choose whether you want to cancel the execution on pending responses, after discussions with Prakash.